src/CFamily.UnitTests/CFamily.UnitTests.csproj

```diff
@@ -30,5 +30,26 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+
+  <ItemGroup Label="HACK to make the C++ rule loader tests work">
```

Comment:
Update - moving these tests from the `Integration.Vsix.UnitTests` project causes intermittent test failures on the CI machine. I think it is down the order the projects are built (`CFamily.UnitTests` needs to be built after `Integration.Vsix` for the tests to pass). I'm testing a fix.